### PR TITLE
Replace phan to psalm. Update dev dependencies. Minor fix phpdoc.

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | -

### CODE REVIEW

File: .github/workflows/static.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ jobs:
 
     env:
       extensions: ast
```
Comment (rustamwin) [inline .github/workflows/static.yml:12]: Not required for psalm

**Review by rustamwin — APPROVED** [review]

File: .github/workflows/static.yml (hunk 1 above)
Comment (vjik) [reply .github/workflows/static.yml:12]: Can you fix in [template](https://github.com/yiisoft/template/blob/master/.github/workflows/static.yml) ?
I'll move the actual config here later.

Comment (rustamwin) [reply .github/workflows/static.yml:12]: [Fixed](https://github.com/yiisoft/template/commit/7d77f1512415e96e008af36f8cf8352a1c039cfd)

Comment (vjik) [reply .github/workflows/static.yml]: Done!

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**